.pre-commit-config.yaml

```diff
@@ -28,3 +28,13 @@ repos:
     - id: detect-missing-init
       args: ['--create', "--track", "--python-folders", "tests,PySDM,examples/PySDM_examples"]
 
+  - repo: local
```

Comment:
would it make sense to fetch the hook code from devops_tests repo? (to make it reusable from PyMPDATA and PyPartMC)?